.github/workflows/ci.yml

```diff
@@ -0,0 +1,89 @@
+name: CI
+
+on:
+  pull_request: {}
+  push:
+    branches: [main]
+
+jobs:
+  tests:
+    name: Format and Tests
+    runs-on: "${{ matrix.os }}"
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        experimental: [false]
```

Comment:
what does `experimental` mean here?

Reply:
if setting it to true, it will run without invalidating the whole workflow if these jobs happen to fail due to an unstable dependency. If that is confusing, I'll remove since it is false here